DEV_GUIDE.md

```diff
@@ -14,28 +14,32 @@ Both are integrated with Maven: the build will fail if some files are not format
 To fix formatting issues from the command line, run the following:
 
 ```sh
-mvn xml-format:xml-format fmt:format
+./mvnw xml-format:xml-format fmt:format
 ```
 
 ## Java Version 
 
 Stargate uses multiple JDKs for its various components, as described in the sections below. 
 
 ### Coordinator node
-The coordinator currently runs on Java 8 due to its backend dependencies. It's important to ensure that you have the correct JDK 8 installed before you can successfully compile the Stargate project. There are a number of versions of JDK 8 and a number of different ways to install them, but not all of them will work successfully with Stargate. For comparison, you can reference the JDK version used in our [CI workflow](.github/workflows/ci-test.yml).
+> NOTE: Coordinator related project is located in the [coordinator/](coordinator) directory.
+
+The coordinator currently runs on Java 8 due to its backend dependencies. It's important to ensure that you have the correct JDK 8 installed before you can successfully compile the Stargate project. There are a number of versions of JDK 8 and a number of different ways to install them, but not all of them will work successfully with Stargate. For comparison, you can reference the JDK version used in our [CI workflow](.github/workflows/apis-v2.yaml).
 
 Download JDK 8 from this link: https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot
 
 Install the JDK and add it to your path. 
 
-For example: if you are using a newer version of MacOS, then you are likely using Z-Shell (zsh) by default. So open your `~/.zshrc` file and add the path there:
+For example: if you are using a newer version of macOS, then you are likely using Z-Shell (zsh) by default. So open your `~/.zshrc` file and add the path there:
 
 ```sh
 export JAVA_HOME="/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home"
 export PATH="$JAVA_HOME/bin:$PATH"
 ```
 
 ### API Services
+> NOTE: APIs related project is located in the [apis/](apis) directory.
+
 The Stargate API services that run externally to the coordinator node are located under the `apis` directory. These services require a more modern JDK in order to take advantage of the latest tools and frameworks. See the [APIs README](apis/README.md) for information on compiling and running the API services including the required JDK. 
 
 (The remainder of these instructions focus on working with the Stargate coordinator node.)
```

Comment:
This is not strictly true, as there is some discussion below of running API services. Perhaps we should just delete this line.

Reply:
Removed, adapted below.